src/data/dsfa_de.json

```diff
@@ -0,0 +1,48 @@
+                "<ul><li><a href='/assets/documents/cwa-datenschutz-folgenabschaetzung-anlage1.pdf' target='_blank'>Anlage 1 - Designentscheidungen bei der Entwicklung der Corona-Warn-App der Bundesrepublik Deutschland</a>",
+                "<li><a href='/assets/documents/cwa-datenschutz-folgenabschaetzung-anlage2.pdf' target='_blank'>Anlage 2 - Technisch-Organisatorische Maßnahmen</a>",
+                "<li><a href='/assets/documents/cwa-datenschutz-folgenabschaetzung-anlage3.pdf' target='_blank'>Anlage 3 - Risikomatrix</a></li>"
+            ]
+        },
+        "previous-version-headline": {
+            "title": "Vorherige Versionen"
+
+        },
+        "previous-version-content": {
+            "textblock": [
+                "Vergangene Versionen der Datenschutz-Folgenabschätzung sowie ihre Anhänge können Sie hier einsehen:",
+                "<ul><li><a href='/assets/documents/cwa-datenschutz-folgenabschaetzung.pdf'>Datenschutz-Folgenabschätzung - Version 2.0</a> vom 10.10.2022</li>",
+                "<ul><li><a href='/assets/documents/cwa-datenschutz-folgenabschaetzung-anlage1a-old.pdf' target='_blank'>Anlage 1a - Designentscheidungen bei der Entwicklung der Corona-Warn-App der Bundesrepublik Deutschland</a>",
+                "<li><a href='/assets/documents/cwa-datenschutz-folgenabschaetzung-anlage1b-old.pdf' target='_blank'>Anlage 1b - Designentscheidungen bei der Entwicklung des European Federation Gateway Service (EFGS)</a>",
+                "<li><a href='/assets/documents/cwa-datenschutz-folgenabschaetzung-anlage1c-old.pdf' target='_blank'>Anlage 1c - Designentscheidungen bei der Entwicklung der Antigen-Schnelltest-Anbindung und Nachweisfunktion</a>",
+                "<li><a href='/assets/documents/cwa-datenschutz-folgenabschaetzung-anlage2.pdf' target='_blank'>Anlage 2 - Technisch-Organisatorische Maßnahmen</a>",
+                "<li><a href='/assets/documents/cwa-datenschutz-folgenabschaetzung-anlage3-old.pdf' target='_blank'>Anlage 3 - Risikomatrix</a>",
+                "<li><a href='/assets/documents/cwa-datenschutz-folgenabschaetzung-anlage4-old.pdf' target='_blank'>Anlage 4 - Risikoanalyse Testing</a>",
+                "<li><a href='/assets/documents/cwa-datenschutz-folgenabschaetzung-anlage5-old.pdf' target='_blank'>Anlage 5 - Risikoanalyse Verifikations-Hotline</a>",
+                "<li><a href='/assets/documents/cwa-datenschutz-folgenabschaetzung-anlage6-old.pdf' target='_blank'>Anlage 6 - Risikoanalyse EFSG</a>",
+                "<li><a href='/assets/documents/cwa-datenschutz-folgenabschaetzung-anlage7-old.pdf' target='_blank'>Anlage 7 - Risikoanalyse PPA & EDUS</a>",
+                "<li><a href='/assets/documents/cwa-datenschutz-folgenabschaetzung-anlage8-old.pdf' target='_blank'>Anlage 8 - Risikoanalyse Nachweisfunktion, Schnelltestpartneranbindung, usw.</a></li></ul>",
```

Comment:
The </ul> here is needed so that the textcaption is not on the same level as the bullet points.